src/Platform/Microsoft.Testing.Platform/CommandLine/PlaltformCommandLineProvider.cs

```diff
@@ -33,6 +33,7 @@ internal sealed class PlatformCommandLineProvider : ICommandLineOptionsProvider
     public const string MinimumExpectedTestsOptionKey = "minimum-expected-tests";
     public const string TestHostControllerPIDOptionKey = "internal-testhostcontroller-pid";
     public const string ExitOnProcessExitOptionKey = "exit-on-process-exit";
+    public const string DotNetTestPipeOptionKey = "dotnet-test-pipe";
```

Comment:
Can you pls refactor a bit this const list and split in 2 group to have server ones clear?

...
// Server mode
ServerOptionKey
PortOptionKey
ClientPortOptionKey
ClientHostOptionKey
DotNetTestPipeOptionKey